### PR TITLE
Adds 'dot' test

### PR DESCRIPTION
**Pull request by vipulchhabra99**

### CODE REVIEW

File: cli/test/test_cmddot.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+class TestDot(PopperTest):
+
+    def test_dot(self):
+
+        wf_dir = tempfile.mkdtemp()
+        runner = CliRunner()
+        git.Git(wf_dir).clone(
+            "https://github.com/popperized/github-actions-demo.git")
+
+        workflow_loc = f'{wf_dir}/github-actions-demo/.github/main.workflow'
```
Comment (JayjeetAtGithub) [inline cli/test/test_cmddot.py:23]: May we can change the name to `workflow_path`.

Comment (vipulchhabra99) [reply cli/test/test_cmddot.py:23]: Yes sure

**Comment by JayjeetAtGithub** [comment]
Since the whole point of adding python tests is removing the bash script tests, we can take this opportunity to keep removing the files for which we are adding unit tests. Like this one and scaffold may be.
What do you think?

**Comment by vipulchhabra99** [comment]
> Since the whole point of adding python tests is removing the bash script tests, we can take this opportunity to keep removing the files for which we are adding unit tests. Like this one and scaffold may be.
> What do you think?

Yes sure, it would be nice to remove them and have better clean codebase.

**Comment by JayjeetAtGithub** [comment]
> > Since the whole point of adding python tests is removing the bash script tests, we can take this opportunity to keep removing the files for which we are adding unit tests. Like this one and scaffold may be.
> > What do you think?
> 
> Yes sure, it would be nice to remove them and have better clean codebase.

Perfect

**Comment by JayjeetAtGithub** [comment]
@ivotron  @vipulchhabra99 LGTM

**Pull request merged**